### ISSUE TITLE
[Parcel2] Stopping parcel serve results in an error

### ISSUE THREAD

**Issue by gregid** [parcel-bundler/parcel#5030]
# 🐛 bug report

Stopping `parcel serve` (with Ctrl+C on Windows) logs an error.

## 🎛 Configuration (.babelrc, package.json, cli command)

Not relevant

## 🤔 Expected Behavior

Should silently stop serving parcel

## 😯 Current Behavior

Instead it stops serving parcel but at the same time it logs an error

```js
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! morpho@0.1.0 app2:start: `parcel serve webapp/src/app2/index.html --dist-dir webapp/dist/app2 -p 1002`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the morpho@0.1.0 app2:start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\MeMe\AppData\Roaming\npm-cache\_logs\2020-08-17T13_19_28_609Z-debug.log
```

## 💁 Possible Solution

Handle the error

## 🔦 Context

Not relevant

## 💻 Code Sample

Not relevant

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-nightly.368
| Node             | v14.5.0
| npm/Yarn         | 6.14.5
| Operating System | Windows 10

**Comment by gregid** [comment]
I have narrowed down the nightly build that introduced the bug.
Last build stopping `parcel serve` without error was:
`2.0.0-nightly.357`
The first where the error has been introduced:
`2.0.0-nightly.358`

**Comment by paulhammond** [comment]
I'm seeing this on linux too. I just created PR #5128 which I think should fix it.

**Comment by paulhammond** [comment]
I believe this was fixed by #5130

**Comment by mischnic** [comment]
Indeed 👍
